static/app/views/performance/trends/changeExplorer.tsx

```diff
@@ -177,12 +214,47 @@ function Header(props: HeaderProps) {
         <ChangeType regression={regression}>
           {regression ? t('Ongoing Regression') : t('Ongoing Improvement')}
         </ChangeType>
-        <TransactionName>{transaction.transaction}</TransactionName>
+        <div style={{display: 'flex', alignItems: 'center', marginBottom: space(3)}}>
```

Comment:
We should make this a styled component as well, we can call it `TransactionNameWrapper` just to tidy this part up